<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderView</class>
 <widget class="QWidget" name="FolderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>534</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string notr="true">FolderViewTopBar</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ActionButton" name="togglePlacesPanelButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string notr="true">CheckableButtonLE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pathBar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string notr="true">PathBar</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ActionButton" name="upButton" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="accessibleName">
            <string notr="true">PathBarButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pathLabel">
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="text">
            <string>[path]</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="pathbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="zoomSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="accessibleName">
         <string>FolderViewSlider</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="zoomSliderSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>3</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="IconButton" name="optionsPopupButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>38</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>38</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>CheckableButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="StyledComboBox" name="sortingComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="accessibleName">
         <string>PanelComboBox</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>A - Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z - A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Size</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Size (desc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oldest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Newest</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ActionButton" name="docViewButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>38</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>38</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>PanelButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ActionButton" name="settingsButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>38</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>38</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>PanelButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="panelRightEdgeSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>2</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="ActionButton" name="exitButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>PanelButtonRE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string notr="true">FolderViewContents</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="accessibleName">
         <string notr="true">FolderViewSplitter</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>6</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="placesPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>0</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>PlacesPanel</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ClickableLabel" name="bookmarksLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="accessibleName">
               <string notr="true">PanelSectionHeader</string>
              </property>
              <property name="text">
               <string>BOOKMARKS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="IconButton" name="newBookmarkButton" native="true">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="accessibleName">
               <string>PlacesPanelButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="BookmarksWidget" name="bookmarksWidget" native="true"/>
          </item>
          <item>
           <spacer name="placesPanelSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ClickableLabel" name="directoriesLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="accessibleName">
               <string notr="true">PanelSectionHeader</string>
              </property>
              <property name="text">
               <string>FILESYSTEM</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="IconButton" name="rootButton" native="true">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="accessibleName">
                 <string>PlacesPanelButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="IconButton" name="homeButton" native="true">
                <property name="minimumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="accessibleName">
                 <string>PlacesPanelButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="TreeViewCustom" name="dirTreeView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="thumbnailGridHolder" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="FolderGridView" name="thumbnailGrid">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>gui/customwidgets/clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>StyledComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/customwidgets/styledcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ActionButton</class>
   <extends>QWidget</extends>
   <header>gui/customwidgets/actionbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FolderGridView</class>
   <extends>QGraphicsView</extends>
   <header>gui/folderview/foldergridview.h</header>
  </customwidget>
  <customwidget>
   <class>TreeViewCustom</class>
   <extends>QTreeView</extends>
   <header>gui/folderview/treeviewcustom.h</header>
  </customwidget>
  <customwidget>
   <class>BookmarksWidget</class>
   <extends>QWidget</extends>
   <header>gui/folderview/bookmarkswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>gui/customwidgets/iconbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bookmarksLabel</sender>
   <signal>clicked()</signal>
   <receiver>FolderView</receiver>
   <slot>toggleBookmarks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoriesLabel</sender>
   <signal>clicked()</signal>
   <receiver>FolderView</receiver>
   <slot>toggleFilesystemView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>-10</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirTreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FolderView</receiver>
   <slot>onTreeViewClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTreeViewClicked(QModelIndex)</slot>
  <slot>toggleBookmarks()</slot>
  <slot>toggleFilesystemView()</slot>
 </slots>
</ui>
